tasks/release_plugin.js

```diff
@@ -99,7 +99,8 @@ module.exports = function (grunt) {
 
         cp.exec(lastTagCommand, {cwd: options.repo}, function (err, lastTag) {
             if (err) {
-                grunt.log.error('Repository does not contain tags');
+                grunt.log.error('Repository does not contain tags, defaulting to version 0.0.0');
+                checkLastCommit('0.0.0')
             }
 
             checkLastCommit(lastTag);
```

Comment:
I made a mistake. Now when I'm trying to run `currentVersion` I can see this output:

`>> Repository does not contain tags, defaulting to version 0.0.0
{"currentVersion":"0.0.1-SNAPSHOT"}
Fatal error: Cannot read property '0' of null`

Could you make a change in your code? Simply run `checkLastCommit(lastTag);` in the else statment.